File: programming files/datetime_convert.java
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class datetime_convert {
    public static void main(String[] args) {
        // string that will be formatted
        String dateStr = "2022-03-17 10:45:30";
        // call imported formatting function
        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
        // create date+time object
        LocalDateTime dateObj = LocalDateTime.parse(dateStr, formatter);
        String formattedDate = dateObj.format(DateTimeFormatter.ofPattern("MM/dd/yyyy HH:mm:ss"));
        // print formatted date and time
        System.out.println(formattedDate);
    }
}

File: programming files/datetime_convert.py
from datetime import datetime

#Create string to store date and time
date_str = "2022-03-17 10:45:30"
#Create date object using date_str
date_obj = datetime.strptime(date_str, '%Y-%m-%d %H:%M:%S')
#Format the date and time
formatted_date = date_obj.strftime('%m/%d/%Y %H:%M:%S')

print(formatted_date)
